### PR TITLE
EAGLE-543 Fix eagle server default config, fix eagle server deve dependency

### PR DESCRIPTION
**Pull request by haoch**
# JIRA
https://issues.apache.org/jira/browse/EAGLE-543

# Changes
* Add loaded application providers brief info log.
* Fix ApplicationProviderClass bug
* Remove duplicated dependency
* Add eagle server request log config

### CODE REVIEW

**Comment by hsaputra** [comment]
Could you add more description about how you are fixing the issue?

File: eagle-core/eagle-app/eagle-app-base/src/main/java/org/apache/eagle/app/service/impl/ApplicationProviderSPILoader.java [hunk 1]
```diff
@@ -82,9 +82,15 @@ private void loadProviderFromClassLoader(ClassLoader jarFileClassLoader, Functio
                 providerConfig.setJarPath(jarFileSupplier.apply(applicationProvider));
                 applicationProvider.prepare(providerConfig, getConfig());
                 registerProvider(applicationProvider);
+                LOG.warn("Loaded {}:{} ({}) from {}",
```
Comment (haoch) [inline eagle-core/eagle-app/eagle-app-base/src/main/java/org/apache/eagle/app/service/impl/ApplicationProviderSPILoader.java]: Add loaded application providers brief info log.

File: eagle-core/eagle-app/eagle-app-base/src/main/java/org/apache/eagle/app/service/impl/ApplicationProviderServiceImpl.java [hunk 2]
```diff
@@ -103,8 +103,8 @@ private void validate() {
                         if (!dependency.isRequired()) {
                             LOG.warn("Unable to load dependency {} -> {}", applicationDesc.getType(), dependency, ex);
                         } else {
-                            LOG.error("Failed to load dependency {} -> {}", applicationDesc.getType(), dependency,ex);
-                            throw new IllegalStateException("Failed to load application providers due to dependency missing",ex);
+                            LOG.error("Failed to load dependency {} -> {}", applicationDesc.getType(), dependency, ex);
```
Comment (haoch) [inline eagle-core/eagle-app/eagle-app-base/src/main/java/org/apache/eagle/app/service/impl/ApplicationProviderServiceImpl.java]: Add dependency details for Dependency Missing Exception

File: eagle-core/eagle-app/eagle-app-base/src/main/java/org/apache/eagle/app/spi/ApplicationXMLDescriptorLoader.java [hunk 3]
```diff
@@ -48,7 +48,7 @@ private String generateXMLDescriptorPath() {
     public ApplicationDesc getApplicationDesc() {
         String descriptorPath = generateXMLDescriptorPath();
         ApplicationDesc applicationDesc = new ApplicationDesc();
-        applicationDesc.setProviderClass(this.getClass());
+        applicationDesc.setProviderClass(this.providerClass);
```
Comment (haoch) [inline eagle-core/eagle-app/eagle-app-base/src/main/java/org/apache/eagle/app/spi/ApplicationXMLDescriptorLoader.java]: Fix ApplicationProviderClass bug

File: eagle-core/eagle-metadata/eagle-metadata-jdbc/pom.xml [hunk 4]
```diff
@@ -39,10 +39,6 @@
             <artifactId>commons-dbcp</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.h2database</groupId>
```
Comment (haoch) [inline eagle-core/eagle-metadata/eagle-metadata-jdbc/pom.xml]: Remove duplicated dependency

File: eagle-jpm/eagle-jpm-mr-history/pom.xml [hunk 5]
```diff
@@ -56,7 +56,7 @@
         </dependency>
         <dependency>
             <groupId>org.apache.eagle</groupId>
-            <artifactId>eagle-jpm-app</artifactId>
+            <artifactId>eagle-app-base</artifactId>
```
Comment (haoch) [inline eagle-jpm/eagle-jpm-mr-history/pom.xml]: Resolve invalid dependecy on `eagle-jpm-app`

File: eagle-server-assembly/src/main/conf/server.yml [hunk 6]
```diff
@@ -13,13 +13,25 @@
...
+# http://www.dropwizard.io/0.7.1/docs/manual/configuration.html
+
 server:
   applicationConnectors:
     - type: http
       port: 9090
   adminConnectors:
     - type: http
       port: 9091
+  requestLog:
```
Comment (haoch) [inline eagle-server-assembly/src/main/conf/server.yml]: Add eagle server request log config

**Comment by haoch** [comment]
@hsaputra thanks for reviewing, and added more description about the changes.

**Pull request closed without merging**